homeassistant/components/fritzbox/strings.json

```
@@ -20,12 +20,12 @@
     },
     "abort": {
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
-      "already_configured": "This AVM FRITZ!Box is already configured.",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "not_found": "No supported AVM FRITZ!Box found on the network.",
```

I would suggest to change this as well
```suggestion
      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
```